src/main/java/wooteco/subway/path/domain/SubwayPath.java

```diff
@@ -13,6 +14,16 @@ public SubwayPath(List<SectionEdge> sectionEdges, List<Station> stations) {
         this.stations = stations;
     }
 
+    public double calculateFare(FarePrincipal farePrincipal) {
+        return farePrincipal.calculateFare(calculateDistance(), findExtraFare());
+    }
+
+    private Long findExtraFare() {
+        return sectionEdges.stream()
+                .mapToLong(SectionEdge::extraFare)
+                .max()
+                .orElse(0);
+    }
```

Comment:
기존 코드에 대한 테스트는 모두 작성하지 못하더라도 새로 작성된 코드들은 테스트가 필요하지 않을까요? 🤔
최소한으로 핵심 비즈니스를 담고 있는 코드들은 테스트가 필수라고 생각해요. 
와일더 생각은 어떤가요?? 

```suggestion
    public double calculateFare(FarePrincipal farePrincipal) {
        return farePrincipal.calculateFare(calculateDistance(), findExtraFare());
    }

    private Long findExtraFare() {
        return sectionEdges.stream()
                .mapToLong(SectionEdge::extraFare)
                .max()
                .orElse(0);
    }
    
```

전체적으로 스페이스 컨벤션도 확인해보면 좋을 것 같아요 :)

Reply:
좋은 지적 감사합니다. 핵심 비즈니스 로직을 수행하는 클래스의 단위 테스트를 추가했습니다.

Reply:
이 클래스는 테스트가 없나요?
추가하신 테스트가 어떤걸까요? 🤔

Reply:
PathFinderTest 에서 테스트를 한다고 생각했는데, SubwayPath 에 대한 테스트도 따로 추가하겠습니다.